projects/js-packages/publicize-components/src/utils/is-nextdoor-connection-supported.ts

```diff
@@ -0,0 +1,10 @@
+import { getJetpackData } from '@automattic/jetpack-shared-extension-utils';
+
+/**
+ * Checks if the Instagram connection is supported.
+ *
+ * @returns {boolean} Whether the Instagram connection is supported
+ */
+export function isNextdoorConnectionSupported() {
+	return !! getJetpackData()?.social?.isNextdoorConnectionSupported;
```

Comment:
Seeing these three like this makes me wonder if these should be combined in some way, there's quite a bit of boilerplate in here

Reply:
Yeah, may be we should pass an array of available connections from the backend.

Reply:
Done ✅